application/migrations/010_add_long_running_flag.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Migration_add_long_running_flag extends CI_Migration
+{
+
+    public function up()
+    {
+        // Add a column to indicate whether the office's crawl takes longer than 23 hours
+        $this->db->query("ALTER TABLE `offices` ADD `long_running` varchar(256) CHARACTER SET latin1 DEFAULT 'FALSE' AFTER `omb_monitored`;");
```

Comment:
You need to use the special ` add_column_if_not_exists` stored proc for this [per the docs here](https://github.com/GSA/project-open-data-dashboard/#migration-requirements) to minimize the changes of a botched migration.

You should also be providing a `down()` which undoes these changes, in case we need to roll back.